util/exec.go

```diff
@@ -21,16 +23,17 @@ type Command struct {
 }
 
 // CommandWithResult executes a command and returns its output, while printing the same output to the given logger.
-func CommandWithResult(logger *log.Logger, command *Command) (output string, err error) {
+func CommandWithResult(ctx context.Context, logger *log.Logger, command *Command) (output string, err error) {
```

Comment:
Please always merge a commit like this earlier. It has now a conflict, 100% avoidable. Nothing special to review.